server/game/core/gameSteps/prompts/ResourcePrompt.ts

```diff
@@ -60,7 +60,8 @@ export class ResourcePrompt extends AllPlayerPrompt {
             menuTitle: promptText,
             buttons: [{ text: 'Done', arg: 'done' }],
             promptTitle: 'Resource Step',
-            promptUuid: this.uuid
+            promptUuid: this.uuid,
+            promptType: 'resource'
```

Comment:
I recommend we create an enum for tracking these, it could go in Constants.ts with the others